src/ark/phenotyping/cluster_helpers.py

```diff
@@ -16,6 +16,38 @@
 from alpineer.misc_utils import verify_in_list
 
 
+def verify_unique_meta_clusters(pixie_remapped_data: pd.DataFrame, meta_cluster_type: str):
```

Comment:
I think it would be helpful to have `meta_cluster_type` specified like Sri has `cluster_type` in another pull request (so it's clear that it should be either "pixel" or "cell"):

```
def save_colored_masks(
        fovs: List[str],
        mask_dir: Union[str, pathlib.Path],
        save_dir: Union[str, pathlib.Path],
        cluster_id_to_name_path: Union[str, pathlib.Path],
        metacluster_colors: Dict,
        cluster_type: Literal["cell", "pixel"],
) 
```